UserDto from communication with frontend.

wac22585 opened the pull request:
Other DTO's have to be implemented at a later point (mostly security dto's)

wac22585 committed:
UserDto from communication with frontend.
Other DTO's have to be implemented at a later point (mostly security dto's)

szh22482 commented on backend/src/main/java/at/spengergasse/backend/dto/UserDto.java:
```diff
@@ -0,0 +1,17 @@
+import java.util.List;
+import java.util.UUID;
+
+public record UserDto (
```
Passwort auch mit übergeben

szh22482 commented on backend/src/main/java/at/spengergasse/backend/dto/UserDto.java:
```diff
@@ -0,0 +1,17 @@
+        Date deletedDate,
+        List<String> roles
+        )
+{ }
```
fromEntity() noch hinzufügen

szh22482 reviewed (changes requested):
Passwort sowie fromEnitity fehlen noch

wac22585 committed:
Added password attribute to enable password changes from the frontend.
The fromEntity method will be used in the service class to convert an entity to a dto.

szh22482 reviewed (approved):
Looks good!
